webapp/src/ts/services/tour.service.ts

```diff
@@ -509,20 +514,34 @@ export class TourService {
     return this.tours.find(t => t.name === name) || this.tours[0];
   }
 
-  private getSettings(name) {
+  private getSkipSettings() {
+    return {
+      reports: {
+        condition: () => this.canViewSidebar,
+        steps:[
+          'mm-form-type-filter',
+          'mm-facility-filter',
+          'mm-date-filter',
+          'mm-status-filter',
+        ]
+      }
+    };
+  }
 
+  private getSettings(name) {
     const settings = this.getTour(name);
+    const skipSettings = this.getSkipSettings()[settings.name];
     settings.autoscroll = false;
+    settings.steps = settings.steps.filter(step => {
+      return !(skipSettings?.condition() && skipSettings.steps.includes(step.element));
```

Comment:
Good points. I think the idea is to someday use another training mechanism and retired the tour, for new development we decided to not use it and keep it just for the old stuff.

I'm choosing a middle point solution: 
1. I'm adding a skip function to the steps we need to skip, so code isn't convoluted
2. I'm going to remove these steps when retrieving the tour in `getTour(name)`. At that point we still have some control over the tour before passing it over to the next process. I'd like to avoid refactoring [bootstrap-tour-standalone.js](https://github.com/medic/cht-core/blob/master/webapp/src/js/bootstrap-tour-standalone.js)

Reply:
Sounds good!  There is no point in completely refactoring something that we plan to phase out.